sw/airborne/modules/ctrl/approach_moving_target.c

```diff
@@ -75,11 +82,25 @@ static void send_approach_moving_target(struct transport_tx *trans, struct link_
 
 void approach_moving_target_init(void)
 {
+  approach_moving_target_set_low_pass_freq(cutoff_freq_filters_hz);
 #if PERIODIC_TELEMETRY
   register_periodic_telemetry(DefaultPeriodic, PPRZ_MSG_ID_APPROACH_MOVING_TARGET, send_approach_moving_target);
 #endif
 }
 
+void approach_moving_target_set_low_pass_freq(float filter_freq) {
+  cutoff_freq_filters_hz = filter_freq;
+  // tau = 1/(2*pi*Fc)
+  float tau = 1.0 / (2.0 * M_PI * cutoff_freq_filters_hz);
+  float sample_time = 1.0 / PERIODIC_FREQUENCY;
```

Comment:
PERIODIC_FREQUENCY is wrong here, this should be the frequency of the `follow_diagonal_approach` (which is 100Hz) you can get that define out of the generated code.